grafana/dashboards/hazl-orders-playground.json

```diff
@@ -0,0 +1,1671 @@
+{
+  "annotations": {
```

Comment:
all other dashboards start with `__inputs`, `__elements`, and `__requires` sections. you might get these for free if you try different grafana export options, or, just crib from an existing dashboard.